src/lib-components/ButtonLink.vue

```diff
@@ -93,8 +93,9 @@ export default {
           return 'svg-arrow-download'
         case isInternalLink(this.to):
           return 'svg-arrow-right'
-        case this.iconName === 'none':
+        case this.iconName === '':
```

Comment:
before this add 
Add another case 
to check the linkTarget
case this.linkTarget === '_blank' || this.iconName === 'svg-external-link':
    return 'svg-external-link'

Reply:
remove 
case this.iconName === '':
 return 'none'
 
 make 
default:
return 'none'